README.md

```diff
@@ -41,51 +31,40 @@ the library are:
 
 * oprof: Optimized code path with compiler flags suitable for use with oprofile.
 
-
 To select a set of methods, you can pass them to configure in one of two ways:
 
-    $ ./configure --with-methods="opt dbg devel"
+    ../configure --with-methods="opt dbg devel"
```

Comment:
If we remove the leading dollar sign, does that also undo the monospace font?

    $ ./test

versus

    ../test

No, looks like it's just the indentation that markdown cares about.  Ok then.

Reply:
Right... the dollar sign was meant to represent a shell prompt, but over the years I've found that it confuses people (do I type a dollar sign?) as much or more than it helps.

Reply:
Also, I used GitHub's in-browser editor (with preview) to make the markdown changes, and I think everything is still going to render correctly.